Studio/src/Application/Optimize/OptimizeTool.ui

```diff
@@ -685,6 +654,34 @@ background: rgb(255, 255, 127)
         </property>
        </spacer>
       </item>
+      <item row="8" column="2">
+       <widget class="QDoubleSpinBox" name="normals_strength">
+        <property name="sizePolicy">
+         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+          <horstretch>0</horstretch>
+          <verstretch>0</verstretch>
+         </sizepolicy>
+        </property>
+        <property name="minimumSize">
+         <size>
+          <width>100</width>
+          <height>0</height>
+         </size>
+        </property>
+        <property name="maximumSize">
+         <size>
+          <width>100</width>
+          <height>16777215</height>
+         </size>
+        </property>
+        <property name="maximum">
+         <double>999999999.000000000000000</double>
+        </property>
+        <property name="value">
+         <double>10.000000000000000</double>
```

Comment:
this is the minimum for normals strength, right? I think we should be able to set this lower than 10.0. 

(I've used normal strength == 0.0 for some testing, but I don't know if this is useful for users)

Reply:
The minimum is zero.  10.0 is just the default value.